automatic/mkdocs/mkdocs.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>mkdocs</id>
+    <title>MkDocs</title>
+    <version>0.16.0</version>
+    <authors>Tom Christie</authors>
+    <owners>chocolatey</owners>
+    <projectUrl>http://www.mkdocs.org</projectUrl>
+    <projectSourceUrl>https://github.com/mkdocs/mkdocs/</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/mkdocs</packageSourceUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/45d7d88c15fb38a55f15ef9ecf39b356cdbadabd/icons/mkdocs.png</iconUrl>
+    <licenseUrl>https://github.com/mkdocs/mkdocs/blob/master/LICENSE</licenseUrl>
+    <docsUrl>http://www.mkdocs.org</docsUrl>
+    <bugTrackerUrl>https://github.com/mkdocs/mkdocs/issues</bugTrackerUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Project documentation with Markdown.</summary>
+    <description>
+MkDocs is a fast, simple and downright gorgeous static site generator that's geared towards building project documentation. Documentation source files are written in Markdown, and configured with a single YAML configuration file.    
+
+## Features
+
+* **Host anywhere:** MkDocs builds completely static HTML sites that you can host on GitHub pages, Amazon S3, or [anywhere](http://www.mkdocs.org/user-guide/deploying-your-docs/) else you choose.
+* **Great themes available:** There's a stack of good looking themes available for MkDocs. Choose between the built in themes: [mkdocs](http://www.mkdocs.org/user-guide/styling-your-docs/#mkdocs) and [readthedocs](http://www.mkdocs.org/user-guide/styling-your-docs/#readthedocs), select one of the 3rd party themes in the [MkDocs wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes), or [build your own](http://www.mkdocs.org/user-guide/custom-themes/).
+* **Preview your site as you work:** The built-in dev-server allows you to preview your documentation as you're writing it. It will even auto-reload and refresh your browser whenever you save your changes.
+* **Easy to customize:** Get your project documentation looking just the way you want it by customizing the theme.
+</description>
+    <tags>mkdocs markdown documentation foss cross-platform cli</tags>
+    <releaseNotes>http://www.mkdocs.org/about/release-notes/</releaseNotes>
+    <dependencies>
+      <dependency id="Python" version="2.6" />
```

Comment:
They'll drop support for Python 2.6 in 1.0: https://github.com/mkdocs/mkdocs/issues/165. But IMHO as long as Python 2.6 is still supported this should be reflected in the Chocolatey Package. Since this is a minimum constraint it's up to the user to decide which of the supported Python versions he want's to use.